src/core/tools/common/profiler.cpp

```diff
@@ -68,11 +68,11 @@ namespace dsn {
         {
             if (caller != nullptr)
             {
-                auto& prof = s_spec_profilers[caller->spec().code];
                 auto code = caller->spec().code;
+                auto& prof = s_spec_profilers[code];
                 if (prof.collect_call_count)
                 {
-                    prof.call_counts[callee->spec().code]++;
+                    prof.call_counts[code]++;
```

Comment:
fixed in https://github.com/qinzuoyan/rDSN/commit/1961ec28cab20607210a3afb26d927fb46a52b1a